### DISCUSSION TITLE
JssImage is using Next Image Component? sitecore-jss/sitecore-jss-nextjs

### DISCUSSION THREAD

**Discussion by isyedakhtar** [Sitecore/jss#1687] [Help]
### Description

The XM foundation repo https://github.com/sitecorelabs/xmcloud-foundation-head is using JssImage from sitecore-jss/sitecore-jss-nextjs to build Image component. Looking at the code, it seems JssImage renders Next Image component which then translates into standard img html tag. 
One of our client is trying to understand if they can do image optimization attributes that Next Image component supports https://nextjs.org/docs/pages/building-your-application/optimizing/images 
Can someone please advise on this

**Comment by art-alexeyenko** [comment]
@isyedakhtar we are currently working on supporting Nextjs/Vercel image optimization out of the box (we are using Sitecore's optimization currently) - so this should be out sometime in near future.
